index.html

```diff
@@ -0,0 +1,603 @@
+              <svg width="24" height="24">
+                <use xlink:href="images/sprite.svg#pen"></use>
+              </svg>
+              <h3>Tution and Fees</h3>
+            </div>
+            <span>______</span>
+
+            <div class="started-block____right-item">
+              <svg width="24" height="24">
+                <use xlink:href="images/sprite.svg#case"></use>
+              </svg>
+              <h3>Review & Rating</h3>
+            </div>
+
+            <span>______</span>
+
+            <div class="started-block____right-item">
+              <svg width="24" height="24">
+                <use xlink:href="images/sprite.svg#book"></use>
+              </svg>
+              <h3>Universoty Facilities</h3>
+            </div>
+
+            <span>______</span>
+
+            <div class="started-block____right-item">
+              <svg width="18" height="18">
+                <use xlink:href="images/sprite.svg#message"></use>
+              </svg>
+              <h3>Community Q&A</h3>
+            </div>
+
+            <span>______</span>
+          </div>
+        </div>
+      </section>
+
+      <section class="blog-block">
+        <div class="blog-block____title">
+          <h2>Latest Blog</h2>
+          <span>____</span>
+        </div>
+        <div class="blog-block____box">
+          <div class="blog-block____box-left">
+            <div class="blog-block____box-background"></div>
+            <p>DESIGN</p>
+            <img width="328" src="/images/blog-block/card-left.png" alt="card-left" />
+            <img src="/images/courses-block/man_1.png" alt="man_1" />
+            <h4>International Conference on Bussiness</h4>
+            <p>Read Now</p>
+          </div>
+
+          <div class="blog-block____box-right">
+            <div class="blog-block____box-background"></div>
+            <p>DESIGN</p>
+            <img width="328" src="/images/blog-block/card-right.png" alt="card-right" />
+            <img src="/images/courses-block/man_2.png" alt="man_2" />
+            <h4>International Conference on Bussiness</h4>
+            <p>Read Now</p>
+          </div>
+
+          <div class="blog-block____box-numbers">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+          </div>
+        </div>
+      </section>
+
+      <form action="#" id="form">
+        <input type="text" placeholder="Name Surname" />
+        <input type="number" placeholder="Phone" />
+        <input type="email" placeholder="Enter your email..." />
+        <button type="submit">Send</button>
+      </form>
+
+      <section class="sources-block">
+        <svg width="53.12px" height="24px">
+          <use xlink:href="images/sprite.svg#union-icon"></use>
+        </svg>
+        <div class="sources-block-item">
+          <h3>Company</h3>
+          <p>About Company</p>
+          <p>Feature Course</p>
+          <p>Careers</p>
```

Comment:
Please use ul list for this elements and next two as well.

Reply:
Done.